include/natalie/gc/allocator.hpp

```diff
@@ -86,7 +83,7 @@ class Allocator {
 
     void unmark_all_cells_in_all_blocks() {
         for (auto block : *this) {
```

Comment:
This makes me wish we had a `Hashmap<T>::keys()` method... :thinking:

Reply:
Happy to add something along those lines in this PR, haven't included it yet since I wasn't sure if that's out of scope?

Reply:
Let's do that in another PR. We can merge your PR and clean it up later. Thanks!